src/main/ImGuiZepEditor.cpp

```diff
@@ -511,4 +523,75 @@ void ImGuiZepEditor::Render(const char* id, const ImVec2& displaySize)
 	ImGui::EndChild();
 }
 
+void ImGuiZepEditor::Clear()
+{
+	m_buffer->Clear();
+}
+
+void ImGuiZepEditor::GetText(std::string& outBuffer) const
+{
+	outBuffer.reserve(GetTextLength());
+	auto& workingBuffer = m_buffer->GetWorkingBuffer();
+	std::copy(workingBuffer.begin(), workingBuffer.end(), std::back_inserter(outBuffer));
+	// copy 1 less character to avoid adding the null terminator into the std::string size.
+	outBuffer.pop_back();
```

Comment:
this is a confusing way to copy, I'm not sure why you want to remove the null terminator, but how is it different than just doing `std::string(buffer.begin(), buffer.end())`

Reply:
DLL boundary mess

This function can't do any allocations.

Reply:
Also it seems to be copying an extra null terminator, hence the pop_back